app/dturn/dturn.js

```diff
@@ -121,6 +122,14 @@ function insertStatementId(discussionId, userId, statementId) {
       },
     },
   })
+
+  // Only run updates if participants or round changes
+  const participants = Object.keys(Discussions[discussionId].Uitems).length
+
+  if (round != Discussions[discussionId].lastRound || participants != Discussions[discussionId].lastParticipants) {
```

Comment:
round (from the updateUinfo) is not related to lastRound.
round - the current  round the current user is updating info on

lastRound - the max round that any user has been to/can go to.
It probably doesn't make sense to do anything except send the current value of lastRound here.
 
can we just call it participants rather than lastParticipants. 

At line ~260 where there's the comment: // first time for this round, need to setup
we should set lastRound (to round), and send an update.

Then at the end of reconstructDiscussion we should set lastRound to Discussion[discussionId].ShownStatements.length but not send an update.